devtools_m1/cypress/integration/CustomerSubscriptionInteractions/steps.js

```diff
@@ -82,7 +78,7 @@ When('I {string} from the general newsletter', function(state) {
   cy.log('Resetting mocks')
   cy.wrap(Mockclient.reset())
 
-  cy.visit('/newsletter/manage/')
+  cy.visit(getCurrentFrontendDomain() + '/newsletter/manage/')
```

Comment:
It looks like you can call a function from a template string.
```suggestion
  cy.visit(`${getCurrentFrontendDomain()}/newsletter/manage/`)
```

Reply:
I wondered about that, but the syntax seemed to indicate not. Glad to find I was wrong, thanks!